core/includes/menu.inc

```diff
@@ -1905,11 +1905,12 @@ function theme_menu_local_actions($variables) {
 function theme_menu_toggle($variables) {
   $output = '';
   if ($variables['enabled']) {
+    $toggle_text = t(check_plain($variables['text']));
```

Comment:
Since this is a bit of an odd use of `t()`, can we please add an inline comment here to explain why we are doing this?

Reply:
@klonos Sure, I've added one.

Reply:
I think this should be reversed so that translators don't get HTML-encoded strings in their translations.

```suggestion
    $toggle_text = check_plain(t($variables['text']));
```

Reply:
Actually this should probably be outside the theme function and be moved up to `system_menu_block_build()`.